src/main/java/eu/dissco/disscomasschedulerservice/service/MasSchedulerService.java

```diff
@@ -136,20 +135,18 @@ private Map<String, JsonNode> getTargetObjects(Set<MasJobRequest> masRequests,
     var specimenTargets = masRequests.stream()
         .filter(masRequest -> masRequest.targetType().equals(MjrTargetType.DIGITAL_SPECIMEN))
         .map(MasJobRequest::targetId)
-        .map(id -> id.replace(DOI_STRING, ""))
         .collect(Collectors.toSet());
     var mediaTargets = masRequests.stream()
         .filter(masRequest -> masRequest.targetType().equals(MjrTargetType.MEDIA_OBJECT))
         .map(MasJobRequest::targetId)
-        .map(id -> id.replace(DOI_STRING, ""))
         .collect(Collectors.toSet());
     var targetMapSpecimens = new HashMap<>(specimenRepository.getSpecimens(specimenTargets));
     var targetMapMedia = new HashMap<>(mediaRepository.getMedia(mediaTargets));
     verifyTargetExists(specimenTargets, targetMapSpecimens, masRequests, failedMasJobRequests);
     verifyTargetExists(mediaTargets, targetMapMedia, masRequests, failedMasJobRequests);
     return Stream.concat(targetMapSpecimens.entrySet().stream(), targetMapMedia.entrySet().stream())
         .filter(e -> e.getValue() != null)
-        .collect(Collectors.toMap(entry -> DOI_STRING + entry.getKey(), Map.Entry::getValue));
+        .collect(Collectors.toMap(Entry::getKey, Map.Entry::getValue));
```

Comment:
Still looks a bit strange to me, I added this because we want to return it as a full DOI. Really the only step where we should use a reduced DOI is the call to the database. I now feel like the stripped DOI is the default. Anyway don't want to block this, you probably looked into it.